service/create/service.go

```diff
@@ -774,6 +774,12 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 		return microerror.Maskf(executionFailedError, fmt.Sprintf("could not get security group '%s' ID: '%#v'", ingressSGInput.GroupName, err))
 	}
 
+	// Return error if the VPC peering CIDR block is nil. As the security group
+	// rules cannot be created without it.
+	if conn.AccepterVpcInfo.CidrBlock == nil {
+		return microerror.Maskf(executionFailedError, fmt.Sprintf("vpc peering connection cidr block is nil for cluster '%s'", key.ClusterID(cluster)))
+	}
+
```

Comment:
During my local testing I had some panics due to the CIDR block being nil. So I think we need to test for this case.

Reply:
Why can this happen? Does the cluster creation fail or is it retried and some other logic succeeds eventually to fix this?

Reply:
I think its due to a problem with the VPC peering connection. There isn't retry logic for the peering connection so the cluster creation will fail. At least on this execution. I'll look into it.

Reply:
Going to keep the current check as its only happened locally but I want to avoid panics. 

I've created #451 to investigate. It could be retry logic is needed for the getting the VPC peering connection details. But I'd like to do that as a separate change.